rocky/tools/models.py

```diff
@@ -266,13 +270,17 @@ class Indemnification(models.Model):
     user = models.ForeignKey("account.KATUser", on_delete=models.SET_NULL, null=True)
     organization = models.ForeignKey(Organization, on_delete=models.SET_NULL, null=True)
 
+    EVENT_CODES = {"created": 900211, "updated": 900212, "deleted": 900213}
+
 
 class OOIInformation(models.Model):
     id = models.CharField(max_length=256, primary_key=True)
     last_updated = models.DateTimeField(auto_now=True)
     data = models.JSONField(null=True)
     consult_api = models.BooleanField(default=False)
 
+    EVENT_CODES = {"created": 900221, "updated": 900222, "deleted": 900223}
```

Comment:
The event codes of the above 3 models are the same?

Reply:
Nice catch, fixed it